pegasus/sites.v3/hourofcode.com/views/learn_carousel.haml

```diff
@@ -1,45 +1,42 @@
+- item_count = 3
 
-  -unless subheading.nil_or_empty?
-    %p!=subheading
+- if items.count > 0
+  - if shuffle
+    - items = items.to_a.shuffle.to_h
+  %h3{:class=>'course-carousel-title', :style=>"margin-bottom:8px"}!= heading
 
-  .learnpage{:style=>'position:relative'}
+  - unless subheading.nil_or_empty?
+    %p!= subheading
 
+  .learnpage{:style=>'position:relative'}
     .carousel-wrapper
       %div{id: id, class: 'carousel course-carousel'}
         .slides
-
-          -count = 0
-          -items.each_pair do |key, tutorial|
-            -url = tutorials.launch_url_for(key,request.host_with_port)
-            -url = url + "?company=#{company}" if company
-            -styling = "display:none" unless count == 0
-            .slide.clear{:style=>styling}
-              .slide-img<
-                %a{:href=>url, :target=>'_blank'}<
-                  %img{:src=>tutorial[:image].sub("/images/", CDO.code_org_url("/images/fit-520/")).sub(".png", ".jpg")}
-              .slide-text
-                %a{:href=>url, :target=>'_blank'}<
-                .slide-tags= tutorial[:age]
-                %h2= tutorial[:name]
-                .slide-tags= tutorial[:platformtext]
-                %p.slide-longdescription= tutorial[:longdescription]
-              %a{:href=>url, :target=>'_blank'}<
-                .btn-carousel-slide=I18n.t(:go)
-            -count += 1
-
-        %a.slide-prev{:href=>'#'}<
-          %span<
-            %span.label= I18n.t(:previous)
-        %a.slide-next{:href=>'#'}<
-          %span<
-            %span.label= I18n.t(:next)
+          - items.each_slice(item_count) do |slice|
+            .slide.clear
+              - slice.each do |key, tutorial|
+                - url = tutorials.launch_url_for(key, request.host_with_port)
+                - url = "#{url}?company=#{company}" if company
+                .action-block-wrapper
+                  .action-block.action-block--one-col.flex-space-between
+                    .slide-img-1
+                      %a{:href=>url, :target=>'_blank'}<
+                        .slide-tags.age-tags= tutorial[:age]
+                      %h2{style: "font-size: 18px;"}
+                        = tutorial[:name]
+                        %img{:src=>tutorial[:image].sub("/images/", CDO.code_org_url("/images/fit-520/")).sub(".png", ".jpg"), style: "margin-top: 10px;"}
+                    .slide-text-1
+                      .slide-tags= tutorial[:platformtext]
+                      %p.slide-longdescription= tutorial[:longdescription]
+                    %a.link-button{:href=>url, :target=>'_blank'}<
+                      =I18n.t(:go)
+        %button.slide-prev{:href=>'#', style: "border:none; background-color: var(--neutral_light);"}<
+          %img{src: "/images/carousel/prev.svg", alt: "previous"}
+        %button.slide-next{:href=>'#', style: "border:none; background-color: var(--neutral_light);"}<
+          %img{src: "/images/carousel/next.svg", alt: "next"}
```

Comment:
You can use `//code.org/images/carousel/next.svg` here instead of `/images/carousel/prev.svg`.